Alignment/OfflineValidation/python/TkAlAllInOneTool/templates/condorTemplate.submit

```diff
@@ -4,7 +4,7 @@ executable = run.sh
 output = condor/condor.out
 error  = condor/condor.err
 log    = condor/condor.log
-requirements = (OpSysAndVer =?= "CentOS7")
+requirements = (OpSysAndVer =?= "AlmaLinux9")
```

Comment:
Is this correct?  If el9 is required, shouldn't the corresponding scripts  be launched on the batch node via singularity(for an el8 env)?

Reply:
>  If el9 is required, shouldn't the corresponding scripts be launched on the batch node via singularity(for an el8 env)?

the vast majority of the lxplus nodes is now el9, so the use case of running from el8 should be small.